cstrml/staking/src/lib.rs

```diff
@@ -135,7 +136,27 @@ impl<AccountId, Balance: HasCompact + Zero> Default for Validations<AccountId, B
     }
 }
 
+
+/// Preference of what happens regarding validation.
+#[derive(PartialEq, Eq, Clone, Encode, Decode, RuntimeDebug)]
+pub struct ValidatorPrefs {
+    /// Reward that validator takes up-front; only the rest is split between themselves and
+    /// nominators.
+    #[codec(compact)]
+    pub guarantee_fee: Perbill,
```

Comment:
Duplicate with `guarantee_fee` in validation?

Reply:
Since we will change the `V-G Graph`, add `TODO` to remove old `Validation` struct